docs/source/tutorials/parsl-rp/parsl-rp.ipynb

```diff
@@ -0,0 +1,276 @@
+    "\n",
+    "basis spherical\n",
+    "  n library cc-pvtz\n",
+    "end\n",
+    "\n",
+    "mp2\n",
+    "  freeze core\n",
+    "end\n",
+    "\n",
+    "task mp2 optimize\n",
+    "\n",
+    "ccsd\n",
+    "  freeze core\n",
+    "end\n",
+    "\n",
+    "task ccsd(t)\n",
+    "\"\"\"\n",
+    "nwchem_input = '{0}/{1}'.format(os.getcwd(), 'mp2_optimization.nw')\n",
+    "with open(nwchem_input,\"w+\") as f:\n",
+    "    f.writelines(input)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "39d40bda",
+   "metadata": {},
+   "source": [
+    "Now, we import parsl Python module in our application and the RadicalPilotExecutor"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "77692350",
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2022-11-22T12:17:57.635805Z",
+     "start_time": "2022-11-22T12:17:57.340827Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import parsl\n",
+    "import radical.pilot as rp\n",
+    "\n",
+    "from parsl.config import Config\n",
+    "from parsl.app.app import python_app, bash_app\n",
+    "from parsl.executors import RadicalPilotExecutor"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1290100d",
+   "metadata": {},
+   "source": [
+    "Set the path of RadicalPilotExecutor configuration file. The rpex.cfg is a configuraion file that describes the resource\n",
+    "distribuation between functions and executables for RadicalPilotExecutor and pass it to Parsl dataflow configuration with RadicalPilotExecutor to load it. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fde1d4f1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "rpex_cfg = 'rpex.cfg'\n",
+    "config = Config(\n",
+    "    executors=[RadicalPilotExecutor(\n",
+    "        rpex_cfg=rpex_cfg, bulk_mode=True,\n",
+    "        resource='local.localhost', login_method = 'local',\n",
+    "        walltime=30, managed= True, cores= 4\n",
+    "    )])\n",
+    "\n",
+    "parsl.load(config)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6a6f2135",
+   "metadata": {},
+   "source": [
+    "Create a simple Parsl `@bashapp` to invoke the `nwchem` task. The `bashapp` requires the type of the task which in this case it is an `MPI` and the number of cpu_processes (4 MPI ranks where each rank takes 1 core) to run on."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ad3e90a9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "@bash_app\n",
+    "def nwchem_mp2_optimization(cpu_processes=2, cpu_process_type=rp.MPI):\n",
+    "\n",
+    "    return \"{0} {1}\".format(nwchem, nwchem_input)\n",
+    "\n",
+    "# invoke the nwchem_mp2_optimization\n",
+    "future  = nwchem_mp2_optimization()\n",
+    "\n",
+    "# wait for the results of the nwchem task.\n",
+    "if future.result() == 0:\n",
+    "        print('Parsl task {0} finished'.format(future.tid))    "
```

Comment:
- does nwchem task produce some output? can we print it out? (and maybe also to print the final status?)
- can we stop a parsl process at the end? (since when I try to reload config with different parameters it fails -> `RuntimeError: Config has already been loaded`)

Reply:
You are only allowed to load the config once I think.